src/App.js

```diff
@@ -5,16 +5,33 @@ import Header from "./components/Header";
 import Hero from "./components/Hero";
 import Movies from "./components/Movies";
 import movieData from "./moviesData";
+import Details from "./components/Details";
 
 function App() {
   const [moviesData, setMoviesData] = useState(movieData.movies);
+  const [selectedMovie, setSelectedMovie] = useState(null);
```

Comment:
Nice work setting the single movie state here! Just a suggestion: it’s recommended to assign a specific data type to the state of your component. For example, if the single movie is an object, the initial state should be an empty object instead of null. Assigning null to a state can obscure the state’s type and make debugging more challenging.